content/blog/pycon-india-2016-workshop-introduction.md

```diff
@@ -0,0 +1,33 @@
+Title: PyCon India 2016 - Workshop Introduction
+Date: 2016-08-17 15:00
+Category: 2016
+Slug: workshop-introduction-2016
+Summary: PyCon India 2016 - Workshop Introduction
+
+
+The workshops at [PyCon India](https://in.pycon.org/2016/) will be held on 23rd September, 2016 and are two and half hour long of hands-on sessions, so be sure not to miss it.
+
+There will be three workshops running parallely in different Rooms, make sure that your interested Workshop(s) do not conflict.
+
+There are few choices when it comes to Workshop:
+
+
+- [Building a Lie Detector: Multi-Modal Sentiment Analysis ](https://in.pycon.org/cfp/2016/proposals/building-a-lie-detector-multi-modal-sentiment-analysis~epk1d/) by Mimansa Jaiswal
+
+- [Docker Workshop](https://in.pycon.org/cfp/2016/proposals/docker-workshop~dyXwb/) by Lalatendu Mohanty
+
+- [Talking to Machines : Optimizing Neural Networks with Theano](https://in.pycon.org/cfp/2016/proposals/talking-to-machines-optimizing-neural-networks-with-theano~ep2rb/) by kumarkrishna
+
+- [Productive Coding with PyCharm ](https://in.pycon.org/cfp/2016/proposals/productive-coding-with-pycharm~aA11a/) by pauleveritt
+
+- [Demystifying the Django REST Framework](https://in.pycon.org/cfp/2016/proposals/demystifying-the-django-rest-framework~dBNxd/) by Haris Ibrahim K. V.
+
+- [Scaling Django with Kubernetes](https://in.pycon.org/cfp/2016/proposals/scaling-django-with-kubernetes~boONb/) by Saket
+
+
+More details about Workshops and Selected talks available at [PyCon India 2016 - Proposals](https://in.pycon.org/cfp/2016/proposals/#proposals)
+
```

Comment:
Further details about workshops and selected talks are available at [PyCon India 2016 - Proposals](https://in.pycon.org/cfp/2016/proposals/#proposals)